pkg/pgmodel/upgrade_tests/upgrade_test.go

```diff
@@ -35,16 +35,18 @@ import (
 )
 
 var (
-	testDatabase = flag.String("database", "tmp_db_timescale_upgrade_test", "database to run integration tests on")
-	useExtension = flag.Bool("use-extension", true, "use the promscale extension")
-	printLogs    = flag.Bool("print-logs", false, "print TimescaleDB logs")
+	testDatabase  = flag.String("database", "tmp_db_timescale_upgrade_test", "database to run integration tests on")
+	useExtension  = flag.Bool("use-extension", true, "use the promscale extension")
+	useTimescale2 = flag.Bool("use-timescale2", false, "use TimescaleDB 2.0")
+	printLogs     = flag.Bool("print-logs", false, "print TimescaleDB logs")
 )
 
 var cleanImage = "timescale/timescaledb:latest-pg12"
 var prevDBImage = "timescale/timescaledb:latest-pg12"
 
 func TestMain(m *testing.M) {
 	var code int
+	// TODO Timescale 2.0 and regular postgres
```

Comment:
what's this todo for?

Reply:
@cevian currently we only test upgrading a DB with timescale-1.X to another DB with timescale-1.X we don't test that upgrades work on vanilla postgres or timescale-2. We need to decide if and how we want to do those